packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -371,6 +470,9 @@ export default class ScriptTransformer {
     if (map) {
       const sourceMapContent =
         typeof map === 'string' ? map : JSON.stringify(map);
+
+      invariant(sourceMapPath, 'We should always have default sourceMapPath');
```

Comment:
I don't think this can ever not be set

Reply:
I add this because `sourceMapPath` [might be set to null](https://github.com/ychi/jest/blob/141b722827615788918e44e0805472f5bec4c90d/packages/jest-transform/src/ScriptTransformer.ts#L482), making the type `Config.Path | null` in this function. 

Another approach might be declaring a new local variable for return?